OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -463,6 +463,35 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </q>
                                     <note> <!-- more --> The end of the note is barely legible due to a stain of water.</note>
                                 </item>
+                                <item xml:id="a2">
+                                    <desc type="Unclear"></desc>
+                                    <locus target="#78va"/>
+                                    <q xml:lang="am">
+                                        <hi rend="rubric">ሐሰበ<supplied reason="omitted">፡</supplied> ፍከሬሁ<supplied reason="omitted">፡</supplied> 
+                                            ለዳዊት<supplied reason="omitted">፡</supplied> ስምአ<supplied reason="omitted">፡</supplied> ሞተ<supplied reason="omitted">፡</supplied></hi>
+                                        ምሕረት<supplied reason="omitted">፡</supplied> ወንጌላዊ<supplied reason="omitted">፡</supplied> ወርህን<supplied reason="omitted">፡</supplied> 
+                                        በምቶታ<supplied reason="omitted">፡</supplied> ፶ግደፍ፡ ደግሞሞ<supplied reason="omitted">፡</supplied> ሐባርን<supplied reason="omitted">፡</supplied> 
+                                        ሰምን፻፶ግደፍ፡ የወጸውን፡ የዳዊቱን
+                                    </q>
+                                    <note></note>
+                                </item>
+                                <item xml:id="a3">
+                                    <desc type=""></desc>
+                                    <locus from="78vb" to="80rb"/>
+                                    <q xml:lang="gez">
+                                        <hi rend="rubric">ተአምረ፡ ዘገ<supplied reason="omitted">ብ</supplied>ረ፡</hi> እግዚእነ፡ ወአምላክነ፡ ኢየሱስ፡ ክርስቶ<supplied reason="omitted">ስ</supplied>
+                                        በኃይለ፡ ሞቱ፡ <sic>ማኀየዊ፡</sic> <add place="above">ይምሐሮ፡ ለ</add>ገብሩ፡ ገብረ፡ ማርያም፡ ወሀሎ፡ ፩ድውይ፡ እንዘ፡ ይግ<add place="above">ዕ</add>ር፡ 
+                                        ወየአውዩ፡ ፲<add place="above">፪</add>ዓመተ፡ ወዘረከቦ፡ በብዝኃ፡ ኃጢአቱ፡ ወተፈትነ፡ በእሳት፡ ከመ፡ ወርቅ፡ ወብሩር፡ መዓልተ፡ ወሌሊተ፡ 
+                                    </q>
+                                    <note></note>
+                                </item>
+                                <item xml:id="a4">
+                                    <desc type="GuestText"></desc>
+                                    <locus from="80va" to="80vb"/>
+                                    <q xml:lang="gez">
+                                    </q>
+                                    <note>Spaces for rubrication left unfilled.</note>
+                                </item>
                                 
                                 <item xml:id="e1">
                                     <desc> On the inner side of the frontboard a white sticker with the name of <persName ref="PRS5782JuelJen"/> printed
```

Comment:
"A white sticker, with the name <persName ref="PRS5782JuelJen"/> printed on it, is glued to the inner side of the front board."

Reply:
On a more general note @thea-m @SusanneHummel I think we should go for "front board" rather than "frontboard"?

Reply:
front board would be correct.
The ligatus describes them as "left" and "right" boards, and I find their reasoning quite convoncing. However, maybe ts too complicated to change it. For me front board is fine as well.

"The place on the left of the centre of a book opened as if to be read. All the components or features of a binding on this side of the book can therefore be described as left (e.g. left board, left endleaves, etc.). This removes any confusion about what is the front board on books written in arabic or latin, for instance."
https://www.ligatus.org.uk/lob/search?search_api_views_fulltext=front+board